transform/transform.go

```diff
@@ -89,6 +92,7 @@ func (ti *transformInstruction) transform(in interface{}, fieldType string, modi
 	}
 	rawValue, err := jsonpath.Get(path, in)
 	if err != nil {
+		level.Debug(log.NewLogfmtLogger(os.Stderr)).Log("Error in JSONPath during transform: %v", err)
```

Comment:
Perhaps we could initialize the logger elsewhere? Not sure how often transform errors occur but could cause a performance penalty if it happens frequently.

Reply:
Maybe? This only happens when the jsonpath lib fails. That only should fail when it can't find the field or the gval library fails. So only when we have fields done incorrectly.